src/components/spaces/organisms/organelle-wrapper.tsx

```diff
@@ -114,19 +114,33 @@ export class OrganelleWrapper extends BaseComponent<OrganelleWrapperProps, Organ
 
   public updateReceptorImage() {
     const model = this.getModel();
-    if (model.world.getProperty("working_receptor")) {
-      model.view.hide("#receptor-broken", true);
-      if (model.world.getProperty("hormone_bound")) {
-        model.view.hide("#receptor-working", true);
-        model.view.show("#receptor-bound", true);
+    const { zoomLevel } = this.props;
+    if (zoomLevel === "cell") {
+      model.view.show(".receptor-mini", true);
```

Comment:
I wonder if ultimately we'll want to see the difference between broken and functional proteins at the cell level? Either way, I agree this is good for now since the receptors look the same at that scale anyway.